pkg/stack/unwind/compact_unwind_table.go

```diff
@@ -150,29 +185,49 @@ func rowToCompactRow(row *UnwindTableRow) (CompactUnwindTableRow, error) {
 	}
 
 	// Return address.
-	if row.RA.Rule == frame.RuleUndefined {
-		rbpType = uint8(rbpTypeUndefinedReturnAddress)
+	switch row.RA.Rule {
+	case frame.RuleOffset:
+		if arch == elf.EM_X86_64 {
+			// fmt.Println("entryyy")
+			// maybe just set RA rule to undefined for x86
+			lrOffset = 0
```

Comment:
Instead of having a special case here, why don't we store the actual value for every architecture? While we won't use it in the x86 unwinder, that way we avoid having a special case here, and perhaps the tests for the compact unwind table will be cleaner too. What do you think?